ScoutSuite/providers/aws/rules/findings/ec2-instance-older-than-expected-date.json

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "EC2 Instance Older than Expected Date",
+    "rationale": "To detect EC2 instances older than an arbitrary date, for example to ensure they haven't escaped regular maintenance and purging",
```

Comment:
```suggestion
    "rationale": "To detect EC2 instances that were launched more than an arbitrary number of days. For example, this can be used to ensure that instances are getting regular maintenance and purging.",
```